Assets/Vimeo/Scripts/Recorder/VimeoPublisher.cs

```diff
@@ -114,23 +118,38 @@ private void VideoUpdated(string response)
             UploadProgress("SaveInfoComplete", 1f);
         }
 
+        private void NetworkError(string error_message)
+        {
+            if (OnUploadFail != null) {
+                OnUploadFail("It seems like you are not connected to the internet or are having connection problems.");
```

Comment:
Maybe rename this to `OnNetworkError` or better yet switch to the event type delegation (#29)

Reply:
Addressed in this commit https://github.com/vimeo/vimeo-unity-sdk/pull/28/commits/056dc50026b3e527bf7f666ccaa9f5a89e05416f